.github/workflows/ci.yml

```diff
@@ -4,12 +4,12 @@ on: [pull_request, push]
 
 jobs:
   main:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
I'm curious: why this change? Do you specifically want an environment that updates itself over time?

Reply:
Yes, exactly. Right now `ubuntu-latest` is `ubuntu-20.04`, but 22.04 is already available and will become `ubuntu-latest` eventually, while 20.04 will disappear. We don't have anything particularly bound to the OS and it should be fine and is unlikely to break if we auto-update.

Reply:
Oh yeah that makes sense!